nekoyume/Assets/_Scripts/UI/Module/Button/ConditionalCostButton.cs

```diff
@@ -48,11 +48,18 @@ private struct CostText
         [SerializeField]
         private List<GameObject> costParents = null;
 
-        private readonly Dictionary<CostType, long> _costMap = new Dictionary<CostType, long>();
-
-        public long CrystalCost => _costMap.TryGetValue(CostType.Crystal, out var cost)
-            ? cost
-            : 0L;
+        private readonly Dictionary<CostType, long> _costMap =
+            new Dictionary<CostType, long>();
+
+        public long CrystalCost =>
+            _costMap.TryGetValue(CostType.Crystal, out var cost)
+                ? cost
+                : 0L;
+
+        public int ArenaTicketCost =>
```

Comment:
Is there a reason why the type is int? I think long is safer.

Reply:
I can't imagine the case to use greater then 2.1b arena tickets  in the `BattleArena` action.😅

Reply:
🤣 OK I agree it!